### DISCUSSION TITLE
关于 Pagination 分页器组件开发相关

### DISCUSSION THREAD

**Discussion by kongyich** [FightingDesign/fighting-design#27] [Ideas]
## 配置项

**基本配置**

`simple`: 是否为简洁模式， 默认为 true （非简洁模式含条数选择器，快速跳转，总页数，上下翻页，简洁模式仅支持上下翻页）
`background `: 设置分页按钮的背景色
`current`: 当前页数
`page-size`: 每页显示的个数
`total `: 总条目数
`default-current`: 默认的页数
`page-count`: 总页数，与 total 二选一
`disabled`: 是否禁用
`ide-single-page`: 只有一页时，是否隐藏
`prev-text`: 自定义上一页按钮的文字，与 prev-icon 二选一
`next-text`: 自定义下一页按钮的文字，与 next-icon 二选一
`prev-icon`: 自定义上一页按钮的icon，与 prev-text 二选一
`next-icon`: 自定义下一页按钮的icon，与 next-text 二选一

`item-style`: 分页按钮的自定义样式
`active-item-style`: 当前选中按钮的自定义样式
`prev-next-style`: 上下翻页按钮样式

`show-item-count`: 显示分页按钮的个数，超出此个数会显示省略号
`show-borders`: 当显示省略号时，是否显示第一和最后一页
`options`: 需传入数组，单独配置其他功能，包含 `sizeOptions`(条数选择器)，`jumper`(快捷跳转)，`allTotal`(总页数)

**事件**
`size-change`: page-size 改变时会触发	返回每页条数
`current-change`: current 改变时会触发	返回当前页
`prev-change`: 用户点击上一页按钮改变当前页后触发	返回当前页
`next-change`	用户点击下一页按钮改变当前页后触发	返回当前页

**Comment by Tyh2001** [comment]
先别急，这个组件复杂度较高，各种配置和设计稿都订好了再开始哈~

**Comment by kongyich** [comment]
不着急～
